securedrop/crypto_util.py

```diff
@@ -356,18 +349,7 @@ def decrypt(self, secret, ciphertext):
         return data.decode('utf-8')
 
 
-def clean(s, also=''):
-    """
-    >>> clean("[]")
-    Traceback (most recent call last):
-      ...
-    CryptoException: invalid input: []
-    >>> clean("Helloworld")
-    'Helloworld'
-    """
-    for c in s:
-        if c not in DICEWARE_SAFE_CHARS and c not in also:
-            raise CryptoException("invalid input: {0}".format(s))
-    # scrypt.hash requires input of type str. Since the wordlist is all ASCII
-    # characters, this conversion is not problematic
-    return str(s)
+def _validate_name_for_diceware(name: str) -> None:
```

Comment:
Why the private naming?

Reply:
To convey the fact that it's only used in this file

Reply:
OK. I'd count test code, but this is fine.